src/index.js

```diff
@@ -266,7 +268,11 @@ function Postgres(a, b) {
         throw e instanceof PostgresError && e.code === '25P02' && uncaughtError || e
       }
 
-      !name && await sql`commit`
+      if (transactionId) {
+        !name && await sql.unsafe(`prepare transaction '${transactionId}'`)
```

Comment:
`transactionId` isn't escaped so users code could easily break with a `transactionId` that contains a `'`. Also, if for some reason a user pass a unsanitized value as input, there's a risk of SQL injection :confused: